application-templates/javascript/event/src/middleware/error.middleware.js

```diff
@@ -1,24 +1,18 @@
 import CustomError from '../errors/custom.error.js';
 
-/**
- * Middleware for error handling
- * @param error The error object
- * @param req The express request
- * @param res The Express response
- * @param next
- * @returns
- */
-export const errorMiddleware = (error, _req, res, _next) => {
+export const errorMiddleware = (error, _req, res) => {
   if (error instanceof CustomError) {
     if (typeof error.statusCode === 'number') {
       res.status(error.statusCode).json({
         message: error.message,
         errors: error.errors,
+        stack: process.env.NODE_ENV === 'development' ? error.stack : undefined,
       });
 
       return;
     }
   }
 
-  res.status(500).send('Internal server error');
+  res.status(500).send('Internal server error: ' + process.env.NODE_ENV === 'development' ? error.stack : undefined);
```

Comment:
Here `error` is `unknown`. We need to check if the `error` object is an instance of `Error` before accessing the `stack` property.

The `process.env.NODE_ENV === 'development'` logic is duplicated.